### ISSUE TITLE
Basic static website with Github Pages?

### ISSUE THREAD

**Issue by 16patsle** [phaser-discord/administration#3]
Just a thought. Invite link and so on

**Comment by jdotjs** [comment]
yup, using the wiki as a place to gather longer form content and tutorials also seems reasonable

**Comment by jdotjs** [comment]
broadly speaking I like some sort of "external face" to the server so that it's possible to slowly work your way into the community. Basically "hey, where can i learn about this phaser stuff" -> "oh, these docs are neat" -> "wow it's an existing community, maybe i should say hi"
